provider/views.py

```diff
@@ -9,6 +8,14 @@
 from . import constants, scope
 
 
+try:
+    from django.http import JsonResponse
```

Comment:
great, I would add since >= django1.7 comment and maybe move that to `provider/compat` ?